dart/lib/src/sentry_options.dart

```diff
@@ -538,6 +539,9 @@ class SentryOptions {
   @meta.experimental
   bool enableLogs = false;
 
+  @meta.experimental
+  SentryLogBatcher logBatcher = NoopLogBatcher();
```

Comment:
m: IMO we should not expose this via an option. Instead I'd rather we just make it an instance of `SentryClient`. This gives us the flexibility to make breaking changes.

Reply:
this or mark it as `@internal`